### ISSUE TITLE
Prepare Ingest for Product Directory Data Mapping

### ISSUE THREAD

**Issue by pjhill** [department-of-veterans-affairs/va.gov-team#45391]
# Description
Given the product attributes that we will be tracking in the new product directory, what preparations can we make to our ingest process in order to be able to map key quality assurance practice artifacts to products via the attributes that we now track?

# Tasks
- [x] Identify areas of the ingest that can be reworked to map to newly tracked attributes in the product directory
- [x] Check into the Domo datasets

# Acceptance Criteria
- [x] Data ingests are functioning without error

**Comment by CBonade** [comment]
Data ingest from vets website to Bigquery:

This was functioning properly and required no changes. All new IDs are correctly being ingested into BigQuery attached to our test results and other relevant data.

Data ingestion from BigQuery into Domo:

These I had mostly changed prior due to email spam of errors that I needed to cut off from the automated scheduled ingest attempting to run. There were a couple more that I tweaked, and a couple more that will need to be tweaked still once we get updated compliance check data running regularly.

QA Standards Compliance Check and Product List Participation metrics:

This, was completely broken and needed an overhaul. There's two pieces of this:
1) The pass/fail measurement against our list of QA Standards. This lives in its own database table that just tracks pass/fail of each product in each category.
2) The check for how many fields, and specifically critically important fields, are being filled out in the product list for each product.

Both of these had a variety of things that no longer worked and overall, the code was able to be cut down significantly from its old method thanks to the product directory being stored as a json now instead of a CSV. That said, as of this morning, both of these are merged and will run on their next scheduled run which should be overnight tonight heading into tomorrow.
